benchmarks/e2e/tests/e2e.go

```diff
@@ -11,6 +11,7 @@ import (
 	// test sources
 	_ "github.com/realshuting/multi-tenancy/benchmarks/e2e/tests/block_cluster_resources"
 	_ "github.com/realshuting/multi-tenancy/benchmarks/e2e/tests/configure_ns_quotas"
+	_ "github.com/realshuting/multi-tenancy/benchmarks/e2e/tests/block_multitenant_resources"
 )
```

Comment:
Can we make these paths relative? Currently these seem to be referring to a forked repo...

Reply:
I will new commit in each pr correcting this ASAP

Reply:
updated